Hai Reveal conversations: reduce text walls in hai reveal 1 intro (#7566)
endless-sky/endless-sky: data/hai/hai reveal 1 intro.txt

			label "skipped tours"
			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall. "Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils. This unprecedented raid in broad daylight has the security forces scrambling to redistribute appropriate resources to handle the threat, and all security personnel on leave have been reactivated until better support can be provided."`
			label "that's awful"
			choice

> 				`	Observe the reaction of the Hai in the room.`

Reviewer: ```suggestion
				`	(Observe the reaction of the Hai in the room.)`
```

> 				`	Ignore the news and move on.`

Reviewer: These actions should have brackets around them. 

I feel like changing the second option to (Move on.) or (Keep walking.) might be better. Ignoring the news appears apathetic, unless the player is a xenophobe.

Author: Could you please try the "suggest changes" feature? It'll make it easier to apply your changes (I can just press a button). To suggest a change, use the button that looks like  folded page with a + and - inside it.

Reviewer: Okay, I wasn't aware of it. I'll try it for the next one.

> 				`	Ignore the news and move on.`

Reviewer: ```suggestion
				`	(Keep walking.)`
```

			`	You haven't really seen very much of the Hai so far, and you can't help but think the size of the detachment is rather militant of them. It's impossible to judge if it's for your safety or whether, given your status as a "war hero", it's for theirs.`
			`	Word of your status in human space also seems to have spread ahead of you, perhaps because you are a significant human visitor in the current climate. A small crowd of Hai gather to peek at you on the short walk from the spaceport to the government building you are led to.`
			label resume
			choice
				`	(Keep silent and observe.)`
					goto karma
				`	(Ask about the large security detachment.)`
			`	You turn to the Hai with the most decorated uniform and ask, "Why do I have so many escorts?"`

> 			`	The Hai responds, "Crowd control," he says. "You've become the center of attention of politicians in a media circus. Life in Hai space is complicated of late, and we want to make sure you are not stopped by crowds with endless questions before reaching your meeting."`

Reviewer: ```suggestion
			`	"Crowd control," the Hai responds. "You've become the center of attention of politicians in a media circus. Life in Hai space is complicated of late, and we want to make sure you are not stopped by crowds with endless questions before reaching your meeting."`
```

			choice

> 				`	Keep silent and observe.`

Reviewer: Same here with the brackets.

> 				`	Keep silent and observe.`

Reviewer: ```suggestion
				`	(Keep silent and observe.)`
```

			`	You haven't really seen very much of the Hai so far, and you can't help but think the size of the detachment is rather militant of them. It's impossible to judge if it's for your safety or whether, given your status as a "war hero", it's for theirs.`
			`	Word of your status in human space also seems to have spread ahead of you, perhaps because you are a significant human visitor in the current climate. A small crowd of Hai gather to peek at you on the short walk from the spaceport to the government building you are led to.`
			label resume
			choice
				`	(Keep silent and observe.)`
					goto observe

> 				`	(Ask about the large security detachment.)`

Reviewer 2: Any choices in parentheses are "action choices," with choices in quotes being "dialog choices." We typically should not have action choices where the action is to say something, such as in this second choice.

Author: I reworked that section, so you actually ask about a security detachment. I applied lots of other changes from @bene-dictator to clean up that area too.

> 				`	(Ask about the large security detachment.)`

Reviewer: ```suggestion
				`	"Why do I have so many escorts?"`
```

Author: That doesn't look right in-game because there isn't anyone to ask. I am pondering how to modify the prior text.

Reviewer: You could mention the Hai officer when the player first sees the detachment, something like "As you arrive on Hai-home, you find that a reasonable security detachment is waiting at your designated landing berth. One Hai with a very decorated uniform comes up to greet you, and tells you that he will escort you to the government chambers."

Author: I changed it so the security detachment suddenly leads you to the left to avoid a big crowd.

> 			`	You turn to the Hai with the most decorated uniform and ask, "Why do I have so many escorts?"`

Reviewer: ```suggestion
			`	The Hai closest to you, who happens to have a very decorated uniform, turns to you and responds. "It's for crowd control. You've become the center of attention of politicians in a media circus."
```

> 			`	"Crowd control," the Hai responds. "You've become the center of attention of politicians in a media circus. Life in Hai space is complicated of late, and we want to make sure you are not stopped by crowds with endless questions before reaching your meeting."`

Reviewer: ```suggestion
			`	He looks back to the crowd for a moment. "Life in Hai space is complicated of late, and we want to make sure you are not stopped by crowds with endless questions before reaching your meeting. It would be... inefficient."`
```

			choice
				`	(Keep silent and observe.)`
				`	"Thank you for your service."`
					goto thank
			label observe
			branch karma
				karma > 1
			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`
				goto main
			label karma

> 			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "honorable" and "reasonable" thrown in amongst the babble.`

Reviewer: Are the Hai speaking in your language? Or are these humans talking? It's an interesting part, though.

Author: It is a Hai talking, but I cannot guess why they're talking in Human Standard. It does seem a bit odd. Either they want you to understand, or they're talking to another human.

Author: I changed it to this:

At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."

Solace is a Free Worlds planet near some pirates, waaaay at the far end of human space.

				goto main
			label thank

> 			`	He nods to you, and returns to his duties: holding back one oncoming Hai reporter after another, and the occasional protester.`

Author: The reporter has changed careers to a journalist, and I've removed the protestors. I can't think of any reason why there would be any.

				has "met remington"
			action
				log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations. He is acting as the Syndicate's representative in helping to deal with the situation regarding the Hai.`

> 			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department."  He gestures to the woman behind him and says, "Let me introduce you to Terry."`

Reviewer: I see where you want to go with removing his rambling, and it could be changed. But I wonder if we're losing some characterisation here. It's Remington's job to make things look good, and you can see he's starting to do his job. Then he decides to cut to the chase instead because he realises that this is a serious issue.

Author: Yes, this ramble probably needs to stay in. I'm not sure how to make it shorter without putting it back though.

> 			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department."  He gestures to the woman behind him and says, "Let me introduce you to Terry."`

Reviewer: ```suggestion
			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department. I'm sure you've heard of my work. I..." He stops suddenly. "You know what, nevermind. We're not here to make small talk, there's more important matters to discuss. I look forward to working alongside you in handling this emerging crisis.`
```
Would this be better? If it were to be kept in, the repetition needs to be removed anyway, unless that's there because he seems to be talking so quickly.

Author: You are correct.

				goto next
			
			label met
			action
				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai.`
			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. He gestures to the woman behind him and says, "Let me introduce you to Terry."`
			label next
			`	"Terry Adrianopoulos. Pleasure to meet you," she says with a broad smile and a slight bow. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She seems at once pleasant and approachable while at the same time being slightly off-putting and uncanny. It's an eerie feeling.`

> 			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, and I really do need to get some fresh air, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently," he says. You think that he may have glanced at Terry as he said that, but you aren't sure if it was just incidental or indicative of something. "Nothing I can discuss in detail unfortunately, you know, with all the NDAs involved, but something I'll certainly enjoy taking a break from.`

Reviewer: ```suggestion
			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently." You catch him take a furtive glance at Terry as he says that, but Remington presses on immediately. "Nothing I can discuss in detail, you know, with all the NDAs involved, but it's something I'll certainly enjoy taking a break from.`
```

I don't know how to stick "I really do need to get some fresh air" back in, but I think this is a little better. Less telling, more for the player to decide if it meant anything.

> 			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, so the two of us will be joining you. There is a political situation in the Syndicate that has caused me quite the headache recently," he says. You think he may have glanced at Terry as he said that, but you're not sure if it was incidental or indicative of something. "Nothing I can discuss in detail, with all the NDAs involved, but something I'll certainly enjoy taking a break from.`

Reviewer: The political situation would be ongoing, I wouldn't change that little bit. We could also keep "fresh air" but move it somewhere else that makes more sense, maybe like "caused me quite the headache recently, and I really do need the fresh air." 

Remington was trying to make a joke here with the "You know, with all the NDAs involved". Grey says that the Syndicate forces merchants to sign a lot of NDAs, but this could be better expanded upon in other places. But the joke doesn't work as well without "you know".

Author: I have reverted this change.

> 			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, so the two of us will be joining you. There is a political situation in the Syndicate that has caused me quite the headache recently," he says. You think he may have glanced at Terry as he said that, but you're not sure if it was incidental or indicative of something. "Nothing I can discuss in detail, with all the NDAs involved, but something I'll certainly enjoy taking a break from.`

Reviewer: ```suggestion
			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, so the two of us will be joining you. There is an ongoing political situation in the Syndicate that has caused me quite the headache recently, and I really do need the fresh air," he says. You think he may have glanced at Terry as he said that, but you're not sure if it was incidental or indicative of something. "Nothing I can discuss in detail, you know, with all the NDAs involved, but something I'll certainly enjoy taking a break from.`
```

			`	"Now, allow me to introduce you to Terry," Remington says while gesturing to the woman behind him.`
				goto next
			label met
			action
				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai.`
			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. He gestures to the woman behind him and says, "Let me introduce you to Terry."`
			label next
			`	"Terry Adrianopoulos. Pleasure to meet you," she says with a broad smile and a slight bow. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She seems at once pleasant and approachable while at the same time being slightly off-putting and uncanny. It's an eerie feeling.`

> 			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, and I really do need to get some fresh air, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently." His eyes momentarily flick to Terry as he says that, but Remington presses on immediately. "It's nothing I can discuss in detail of course, with all the NDAs involved you know, but something I'll certainly enjoy taking a break from.`

Reviewer: This is pretty wordy now. 
```suggestion
			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, and I really do need to get some fresh air, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently." His eyes momentarily flick to Terry as he says that, but Remington presses on immediately. "It's nothing I can discuss in detail, with all the NDAs involved, you know, but something I'll certainly enjoy taking a break from.`
```

			label met
			action
				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai.`
			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. "You do remember me, right? I don't want this to be an awkward situation where only I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis.`
			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. He gestures to the woman behind him and says, "Let me introduce you to Terry."`
			label next

> 			`	"Now, allow me to introduce you to Terry," Remington says while gesturing to the woman behind him.`

Reviewer 2: By moving this line above the `next` label, the part of the branch where you haven't met Remington before now doesn't have Remington introducing Terry. So either this line shouldn't be merged with the one before it, or you should duplicate this line to go above the `goto next` line so that both branches properly introduce Terry.

Author: I reworked this area. If you don't know Remington, you get the full text. If you know Remington, then you get to skip a bit. With the other changes to the scene's dialog, the text wall isn't so bad now.

> 			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently." You catch him take a furtive glance at Terry as he says that, but Remington presses on immediately. "Nothing I can discuss in detail, you know, with all the NDAs involved, but it's something I'll certainly enjoy taking a break from.`

Reviewer 2: This mention of the "internal political situation" now feels to me like it comes out of nowhere without the "I really do need to get some fresh air" line. Or at least it makes less sense to me for him to bring it up the way he does.

Author: The fresh air is back in there after two mouse clicks of @bene-dictator's suggested change.

> 			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently." You catch him take a furtive glance at Terry as he says that, but Remington presses on immediately. "Nothing I can discuss in detail, you know, with all the NDAs involved, but it's something I'll certainly enjoy taking a break from.`

Reviewer: ```suggestion
			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, and I really do need to get some fresh air, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently." You catch him take a furtive glance at Terry as he says that, but Remington presses on immediately. "Nothing I can discuss in detail, you know, with all the NDAs involved, but it's something I'll certainly enjoy taking a break from.`
```
I like the fresh air bit but I didn't like where it was. But I can see how the next bit is left hanging.

	on offer
		conversation
			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
			choice

> 				`	Keep silent and let the meeting begin.`

Reviewer: ```suggestion
				`	(Keep silent and let the meeting begin.)`
```

					goto "keep silent"

> 				`	"I don't recall there being nearly so many pirates before."`

Reviewer: I'm wondering where the player is in the situation. Are they already at the meeting? It seems like it in the next part, but there's no ques before that.

Author: There needs to be text mentioning you entered a meeting room.

Author: I have updated it so you are in a meeting room with a fancy holoprojector.
[Friction Diction~3025-05-29 just before leaks response 3.txt](https://github.com/endless-sky/endless-sky/files/9995633/Friction.Diction.3025-05-29.just.before.leaks.response.3.txt)

			`	A functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. Everyone takes their seats at an semicircle-shaped table, and the functionary motions you to the middle seat. A massive holoprojector in front of the table turns on.`
			choice
				`	"I don't recall there being nearly so many pirates before."`
			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well, it seems that the rumors making their way through your media have attracted every opportunist this side of the Deep."`
			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
			`	Alondo looks surprised, but Mr. Yang and Ms. Adrianopolous both merely nod.`
			`	"Extra security forces are being called in, but we've been busier than usual in Wah Ki with our northern brethren."`
			`	Mr. Yang interrupts, "Well then, I'm afraid I may have some bad news. May I?`

> 			`	"One of the things I brought with me was Republic Intelligence's estimate of the effectiveness of various information containment measures." At his word, the holoprojector shows a series of charts.`

Reviewer 2: The "May I?" feels unnecessary now. The previous writing had the line, "He gestures to a display and then takes control when indicated." That "when indicated" part was the answer to the question of "May I?", but now that question gets no answer.

Author: I reworked that section.

			choice

> 				`	(Examine the charts.)`

Reviewer 2: Avoid what I call "empty choices," which are choice branches were you're only given a single action choice that could just as easy be replaced with `(Continue.)`. If this is here because the text at this point is too long, then either try to reduce the text more or make the inclusion of a choice here to break up the text more natural, e.g. by having the player be able to comment on something, even if it all leads to the same place. A few lines up in this same mission there's a choice branch with only a single *dialog* choice for the player, which isn't ideal, but I'd say it's better than a single action choice.

Reviewer: This doesn't seem very different to the player watching the monk's video during the Free Worlds campaign, but both can be improved on.

Reviewer 2: I intentionally left that like that because I know in the future I'll be revisiting and redoing most of it anyway.

Author: Now the player asks, "How much longer can the Republic keep up their side of the ruse?"

The charts come up automatically when Mr. Yang says the words "bad news."

			`	A functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. Everyone takes their seats at an semicircle-shaped table, and the functionary motions you to the middle seat. A massive holoprojector in front of the table turns on.`
			choice
				`	"I don't recall there being nearly so many pirates before."`
			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well, it seems that the rumors making their way through your media have attracted every opportunist this side of the Deep."`
			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
			`	Alondo looks surprised, but Mr. Yang and Ms. Adrianopolous both merely nod.`
			`	"Extra security forces are being called in, but we've been busier than usual in Wah Ki with our northern brethren."`

> 			choice
> 				`	"How much longer can the Republic keep up their side of the ruse?"`
> 			`	Mr. Yang responds, "On that, I'm afraid I may have some bad news." As soon as he says, 'bad news,' the holoprojector shows a series of charts.`

Reviewer 3: Don't like the single choice here. Suggestion?
```suggestion
			choice
				`	"Are the other Hai likely to be a problem here?"`
				`	"How much longer can the Republic keep up the ruse?"`
					goto badnews
			`	Sayari shakes her head. "No problem remotely so severe as the one at hand. Now, I expect you have something for us Mr. Yang?"`
			`	"Well, I'm afraid I may have some bad news from our side." He taps on his screen as he says, 'bad news,' and the holoprojector begins showing a series of charts.`
				goto charts
			label badnews
			`	Mr. Yang responds, "On that, I'm afraid I may have some bad news." He taps on his screen as he says, 'bad news,' and the holoprojector begins showing a series of charts.`
			label charts
```

			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
			`	You get down to business when Sayari inquires how your journey went.`
			choice

> 				`	Keep silent and let the meeting begin.`

Reviewer: It's strange to me that saying nothing skips dialogue, and it's a bit rude that the player can ignore Sayari's question. 

I only noticed now that Sayari also gives important information here about the pirates here, and it's weird that you never get the opportunity to tell this to Teneep before the stuff with Scar's Legion, because otherwise the player has to go to fight pirates to get Teneep to know what the player knew already.

Author: Teeneep already knows about the pirate activity in Hai space. (It is common knowledge.) She doesn't know about Scar's Legion specifically. That mission sequence sets up her strong reaction to the Elder's offer to ship Hai weapons to the humans.

Reviewer: I meant the "rewards" the Elders gave pirates for keeping the secret contained. The strong reaction is good, but I didn't know why the player couldn't have just pointed this information out earlier, something like "Sayari said that the Elders have some arrangement with the pirates. Do you know what this means?"

Author: I meant to take out the "You get down to business..." line. It is gone now, and the changes after it make more sense now.

Author: I want to limit this PR to dialog cleanup only, and your suggested dialog connection (telling Teeneep what Sayari said) is a story change, so it belongs in a different PR.

Reviewer: Okay, that's good.

			`	A functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. Everyone takes their seats at an semicircle-shaped table, and the functionary motions you to the middle seat. A massive holoprojector in front of the table turns on.`
			choice
				`	"I don't recall there being nearly so many pirates before."`
			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well, it seems that the rumors making their way through your media have attracted every opportunist this side of the Deep."`
			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
			`	Alondo looks surprised, but Mr. Yang and Ms. Adrianopolous both merely nod.`
			`	"Extra security forces are being called in, but we've been busier than usual in Wah Ki with our northern brethren."`
			choice
				`	"Are the other Hai likely to be a problem here?"`
				`	"How much longer can the Republic keep up the ruse?"`
					goto badnews

> 			`	Sayari shakes her head. "No problem remotely so severe as the one at hand. Now, I expect you have something for us Mr. Yang?"`

Reviewer: ```suggestion
			`	Sayari shakes her head. "No problem remotely so severe as the one at hand. Now, I expect you have something for us, Mr. Yang?"`
```

				`	"Am I allowed to know more?"`
			`	Alondo gives you a quizzical look. "Of course you are, <last>."`
			label about
			`	He pulls out a hand tablet and shows you a title screen which doesn't automatically look familiar. As he talks he flicks through a series of reference images.`
			`	"There's an older TV show franchise with a cult following called 'Our Fuzzy Paradise' with a whole cast of anthropomorphic squirrels. The production of the first series was entirely funded through a government grants program, as a cover for accidental leaks about the Hai. Xilin tells me that since then it's pretty much run itself.`

> 			`	"We're going to make a tourist trap type theme park destination, and advertise it with a big-budget movie production of the show. We've found a Seerwood producer who'll spoil a 'big surprise' release, and we're passing off the footage as leaked promotional material. For the sake of the cover, we've also hired a 'fan community' to support the production for consistency and accuracy."`

Reviewer 2: RIP "Fellowship is Glamorous."

Author: I put back in the full description, but I let the player interrupt and skip everything after the first paragraph. That's the same way I'm dealing with Turner's movie rambling in my other PRs.  (I hadn't thought of that approach yet when I was reviewing part 1.)

				`	"Am I allowed to know more?"`
			`	Alondo gives you a quizzical look. "Of course you are, <last>."`
			label about
			`	He pulls out a hand tablet and shows you a title screen which doesn't automatically look familiar. As he talks he flicks through a series of reference images.`
			`	"There's an older TV show franchise with a cult following called 'Our Fuzzy Paradise' with a whole cast of anthropomorphic squirrels. The production of the first series was entirely funded through a government grants program, as a cover for accidental leaks about the Hai. Xilin tells me that since then it's pretty much run itself.`

> 			`	"We're going to make a tourist trap type theme park destination and advertise it with a big-budget movie production of the show. We've found a Seerwood producer who'll spoil a 'big surprise' release, and we're passing off the footage as leaked promotional material. For the sake of the cover, we've also hired a 'fan community' to support the production for consistency and accuracy."`

Reviewer: ```suggestion
			`	"We're going to make a tourist trap type theme park destination, and advertise it with a big-budget movie production of the show. We've found a Seerwood producer who'll spoil a 'big surprise' release, and we're passing off the footage as leaked promotional material. For the sake of the cover, we've also hired a 'fan community' to support the production for consistency and accuracy."`
```
